packages/flutter_test/lib/src/controller.dart

```diff
@@ -217,6 +316,309 @@ class SemanticsController {
 
     return false;
   }
+
+  /// Performs the given [SemanticsAction] on the given [SemanticsNode]
+  void performAction(SemanticsNode node, SemanticsAction action, {Object? args, bool checkForAction = true}) {
+    if(checkForAction && !node.getSemanticsData().hasAction(action)){
+      throw StateError(
+        'The given node does not support the $action SemanticsAction.\n'
+        'Found Node: $node');
```

Comment:
this phrasing is kinda odd since the node was given as an input and not "found".

Reply:
Yeah, the `performAction` method originally also "found" the node, but I had to pull that logic out eventually. This was a holdover from that.

Good catch, I just removed the "Found".